README.md

```diff
@@ -1 +1,51 @@
-# logger
\ No newline at end of file
+<img src="https://raw.githubusercontent.com/Yoonit-Labs/android-yoonit-camera/development/logo_cyberlabs.png" width="300">
+
+# logger
+
+Logger is a Logrus extension to quickly use as a Singleton when it is an advantage to your project.
```

Comment:
Is it and extension or a wrapper?

Besides, I would get rid of this "when it is an advantage to your project" part.

I would write, actually: "*Logger is an opinionated wrapper around Logrus that is extremely simple to use.*"

Reply:
**And** "It also has automatic integration with Sentry".